game/ato/flightplans/waypointbuilder.py

```diff
@@ -168,10 +165,7 @@ def hold(self, position: Point) -> FlightWaypoint:
             "HOLD",
             FlightWaypointType.LOITER,
             position,
-            # Bug: DCS only accepts MSL altitudes for the orbit task and 500 meters is
```

Comment:
I don't think this ever got filed. Could you preserve the comment, or file the bug?

Reply:
The bug is filed as #3324 .